.github/workflows/ci.yml

```diff
@@ -156,6 +156,7 @@ jobs:
       contents: write
       id-token: write
       attestations: write
+      packages: read
```

Comment:
issue: Apparently, the packages API does not support authentication using a GITHUB_TOKEN
> GitHub Packages only supports authentication using a personal access token (classic). For more information, see [Managing your personal access tokens](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).

https://docs.github.com/en/packages/learn-github-packages/about-permissions-for-github-packages#about-scopes-and-permissions-for-package-registries

@Julian could you please create a personal access token with `packages: read` and add it to the secrets when you have time? The name of the secret is up to you (I can suggest something like _PACKAGES_TOKEN_ or _READ_PACKAGES_PAT_)

Reply:
Sure, but it looks like you need to add me to the organization first (I tried to do the transfer but got an error)
![image](https://github.com/user-attachments/assets/2f7fbeb3-a711-4444-ba66-f18d2f402a69)

Reply:
All works now! Thank you